projects/plugins/jetpack/extensions/blocks/wordads/test/edit.js

```diff
@@ -47,21 +47,17 @@ describe( 'WordAdsEdit', () => {
 		return { placeholder, selectedFormat };
 	};
 
-	test( 'renders wrapper with correct css class', () => {
+	test( 'matches snapshot', () => {
```

Comment:
This test is brittle since tightly coupled to the markup. Instead, let's use snapshot testing to ensure the UI doesn't change inadvertently.